DEVELOPMENT.md

```diff
@@ -114,6 +114,7 @@ To setup integration tests you will need to export the following environment var
 ```
 # !!!WARNING!!! The integration tests will use these keys to do destructive changes.
 # Never use keys for an organization that contains anything important.
+# The client test data isn't sent to staging by default, use a different environment apart from staging
```

Comment:
Note about using a compatible testing environment apart from staging

Reply:
Not sure this is needed. "External" contributors won't use staging to run the integration tests even if there was an option to send data to this environment. I suppose this is more of an internal note?